tests/integration/src/test/java/org/apache/pulsar/tests/integration/suites/PulsarTestSuite.java

```diff
@@ -60,10 +62,19 @@ protected PulsarClusterSpecBuilder beforeSetupCluster(String clusterName, Pulsar
                 .withCreateContainerCmdModifier(createContainerCmd -> createContainerCmd
                     .withName(kafkaServiceName)
                     .withHostName(clusterName + "-" + kafkaServiceName)));
+
         final String cassandraServiceName = "cassandra";
         externalServices.put(
             cassandraServiceName,
             new CassandraContainer(clusterName));
+
+        // use mySQL for jdbc test
+        final String jdbcServiceName = "jdbc";
+        externalServices.put(
+            jdbcServiceName,
+            new MySQLContainer()
```

Comment:
attach clusterName to the hostname

```
new MySQLContainer()
     .withNetworkAliases(jdbcServiceName)
     .withCreateContainerCmdModifier(createContainerCmd -> createContainerCmd	                
                    .withName(jdbcServiceName)
                    .withHostName(clusterName + "-" + jdbcServiceName)));
```

Reply:
Thanks, this is already done in PulsarCluster